tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```diff
@@ -70,10 +70,61 @@ Services:
         duration: 15
   Tests:
   - Test: jsonsend
-    Arguments:
-    - ""
+    Arguments: &core-document-sizes
+    - --size 10 --parallel 64 --backend-type blobs
+    - --size 1000 --parallel 64 --backend-type blobs
     TestNames:
       Java: jsonsend
+  - Test: jsonreceive
+    Arguments: *core-document-sizes
+    TestNames:
+      Java: jsonreceive
+  - Test: xmlsend
+    Arguments: *core-document-sizes
+    TestNames:
+      Java: xmlsend
+  - Test: xmlreceive
+    Arguments: *core-document-sizes
+    TestNames:
+      Java: xmlreceive
+  - Test: binarydatasend
+    Arguments:
```

Comment:
This is a lot of arguments.  If this exceeds the 6-hour pipeline runtime limit, this might need to be trimmed down.  I am planning to add parallelization to the perf pipelines, but first at the level of running distinct tests in parallel.  Running arguments in parallel is also under consideration but slightly more complex.

Once test parallelization is implemented, an option would be to split these arguments across multiple "tests" (at the perf framework level), which map to the same "test" in Java source code.  One easy split could be `binarydatasend-netty` and `binarydatasend-okhttp`.

Reply:
It should fit into 6hrs. The run with 50% of this takes ~2hrs.
If this https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1685693&view=results completes within timeout I'll leave it as is.

The tricky part here is that sometimes we want to compare netty vs okhttp (and later other clients comparisons). So the split per job would be by (scenario, size) likely and then all http clients on one agent.